commandhandlers.go

```diff
@@ -113,9 +124,10 @@ var commandsGlobal = []Command{
 
 	{
 		Data: api.CreateCommandData{
-			Name:        "ban",
-			Description: "Unverify a user and block their email from verifying",
-			Type:        discord.ChatInputCommand,
+			Name:                     "ban",
+			Description:              "Unverify a user and block their email from verifying",
+			Type:                     discord.ChatInputCommand,
+			DefaultMemberPermissions: permissionRequireBanMembers(),
```

Comment:
This seems inconsistent with the README, I think this should be something like `&(discord.PermissionAdministrator | discord.PermissionBanMembers)`

Reply:
`discord.PermissionAdministrator` grants access to every permission, including `discord.PermissionBanMembers`, so it's not required and would have the same effect. 

From the [Permission Flags](https://discord.com/developers/docs/topics/permissions#permissions-bitwise-permission-flags) documentation:

> Allows all permissions and bypasses channel permission overwrites

Reply:
Alright, this is fine this 👍